.github/actions/release-please-semvar/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Release please
+description: Runs release please and tags unpinned major and minor versions
+runs:
+  using: "composite"
+  steps:
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow
+    # For why we need to generate a token and not use the default
+    - name: Generate token
```

Comment:
did you want to use that action supported by Github here for making a token>

Reply:
done